test.py

```diff
@@ -9,8 +11,17 @@
 
 print oc.toRadString()
 
+
+genday = Gendaymtx(v=True,h=True)
+genday.inputFiles =[r'C:\Ladybug\annual\annualSimulation\State_College__Penn_State___Su_PA_USA.wea'] #This path needs to be present physically !
```

Comment:
@sariths thanks! You're right. It's a bad design to have inputFiles public in some commands and private in some. I was thinking that you need to have the freedom for cases like oconv to assign the input file but I think I should do it in a different way. I added an issue for this: https://github.com/ladybug-analysis-tools/honeybee-core/issues/25